### ISSUE TITLE
Adjust the background color to improve readability

### ISSUE THREAD

**Issue by RVA2869** [govdirectory/website#411]
### The feature

Discussed on #348 

### Why?

Improved readability for web accessibility.

**Comment by Abbe98** [comment]
Screenshots:

![Image](https://github.com/user-attachments/assets/1287b273-8df3-4f8f-9503-e01f16564581)
![Image](https://github.com/user-attachments/assets/9df886c7-8cd0-4a61-9d86-e0d2f16a3fbf)

**Comment by RVA2869** [comment]
I was just thinking about this issue... Wouldn't it be better to have a high-contrast mode? 
Because the dark orange doesn't match so well with the blue. (Just a thought)

**Comment by Abbe98** [comment]
That idea crossed my mind too, I think we should strive towards having better defaults but both the blue and orange would need to be updated and it would be a lot of work(CSS, icons, branding repo, presentation templates, etc). High-contrast mode is more bang for the buck.

**Comment by EoinHo** [comment]
I would be happy to take a look at this.  CSS & WCAG are with my remit.

**Comment by EoinHo** [comment]
I've forked and tweaked as per suggestions above.  A subtle difference but ticks a few boxes.

I tried [colourcontrast.cc](https://colourcontrast.cc/?background=b30704&foreground=fdfdfd) and managed to get AAA normal by pushing the envelope.

What do folks think?

**Comment by EoinHo** [comment]
Sorry, another observation:
On the home page we have the secondary colour (redish) appearing both on a dark background, with white text and with black text.  I suspect it will be very difficult to find a colour in this circumstance that meets WCAG approval.
